src/main/webapp/app/instructor-dashboard/instructor-dashboard-archive-dialog.component.html

```diff
@@ -1,13 +1,13 @@
 <form name="archiveForm" role="form" (ngSubmit)="confirmArchive(exercise.id)" #archiveForm="ngForm">
     <div class="modal-header">
-        <h4 class="modal-title" jhiTranslate="entity.archive.title">Confirm archive operation</h4>
+        <h4 class="modal-title" jhiTranslate="entity.exportRepos.warning">Confirm Export</h4>
         <button type="button" class="close" data-dismiss="modal"
                 (click)="clear()">&times;</button>
     </div>
     <div class="modal-body">
         <jhi-alert-error></jhi-alert-error>
         <p jhiTranslate="instructorDashboard.archive.question"
-           translateValues="{exerciseTitle: '{{exercise.title}}', courseTitle: '{{exercise.course.title}}'}">Confirm archive
+           translateValues="{exerciseTitle: '{{exercise.title}}', courseTitle: '{{exercise.course.title}}'}">Enter all ids of students you want to download the repo from
```

Comment:
I think this is a mistake and you accidentally changed this file. Please revert it.